stan/math/prim/fun/scalbn.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef STAN_MATH_PRIM_FUN_SCALBN_HPP
```

Comment:
Oh this one is like logb. Are these use internally in some weird functions somewhere?